docker/ingress/Dockerfile

```diff
@@ -22,6 +23,7 @@ RUN /bin/bash /tmp/build-nginx.sh
 COPY feed-ingress /
 COPY nginx.tmpl /nginx/
 RUN chown nginx:nginx /nginx/nginx.tmpl
+RUN setcap "cap_net_bind_service=+ep" /usr/sbin/nginx
```

Comment:
Are you sure this works? Don't you need to do this at runtime?

Reply:
feed-ingress pods will need to add `NET_BIND_SERVICE` capability - we'll add a note to the README

Reply:
Ok.